From 5ce8bb2f4c2e578da0231c8ae3d0f72523f4a452 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <baimingcong@uniontech.com>
Date: Fri, 28 Jun 2024 16:23:20 +0800
Subject: [PATCH 113/164] DEEPIN: net: ethernet: phytium: add a missing
 declaration for *np
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix a compilation error:

drivers/net/ethernet/phytium/phytmac_platform.c: In function ‘phytmac_plat_probe’:
drivers/net/ethernet/phytium/phytmac_platform.c:98:55: error: ‘np’ undeclared (first use in this function); did you mean ‘up’?
   98 |                 match = of_match_node(phytmac_dt_ids, np);
      |                                                       ^~
      |                                                       up

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>

Link: https://github.com/deepin-community/kernel-rolling/commit/91e313fa9e3fef7d238bbb0746731306e9a01d14
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/phytium/phytmac_platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/phytium/phytmac_platform.c b/drivers/net/ethernet/phytium/phytmac_platform.c
index 797909211fed..343e1a5ce564 100644
--- a/drivers/net/ethernet/phytium/phytmac_platform.c
+++ b/drivers/net/ethernet/phytium/phytmac_platform.c
@@ -77,6 +77,7 @@ static int phytmac_get_phy_mode(struct platform_device *pdev)
 static int phytmac_plat_probe(struct platform_device *pdev)
 {
 	const struct phytmac_config *phytmac_config = &phytium_1p0_config;
+	struct device_node *np;
 	struct resource *regs;
 	struct phytmac *pdata;
 	int ret, i;
-- 
2.47.0

